src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_SIZE = 10;
+    private final ColorSupplier colorSupplier;
+
+    {
+        colorSupplier = new ColorSupplier();
+    }
```

Comment:
If you want to use init block, place it after fields but better init this in constructor